docs/src/examples/components/Menu/Types/MenuExampleWithSubmenu.shorthand.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { Menu, Provider } from '@stardust-ui/react'
+
+const items = [
+  {
+    key: 'editorials',
+    content: 'Editorials',
+    menu: {
+      items: [
+        { key: '1', content: 'item1' },
+        {
+          key: '2',
+          content: 'item2',
+          menu: { items: [{ key: '1', content: 'item2.1' }, { key: '2', content: 'item2.2' }] },
+        },
+        {
+          key: '3',
+          content: 'item3',
+          menu: { items: [{ key: '1', content: 'item3.1' }, { key: '2', content: 'item3.2' }] },
+        },
+      ],
+    },
+  },
+  {
+    key: 'review',
+    content: 'Reviews',
+    menu: {
+      items: [
+        { key: '1', content: 'item1' },
+        {
+          key: '2',
+          content: 'item2',
+          menu: { items: [{ key: '1', content: 'item2.1' }, { key: '2', content: 'item2.2' }] },
+        },
+      ],
+    },
+  },
+  { key: 'events', content: 'Upcoming Events' },
+]
+
+const MenuExampleWithSubMenu = () => (
+  <Provider
+    theme={{
+      componentStyles: {
+        Menu: {
+          root: {
+            zIndex: 1000,
```

Comment:
shouldn't it be part of the default styles? like

```
// menuItemStyles
'& .ui-menu {
  z-index: ..
}'
```

Reply:
I thought this was added only because of the docs experience, but it turns out that it is common practice to add z-index on the submenus. Will add this styles in the MenuItem, thanks!